modules/mono/mono_gd/gd_mono_method.cpp

```diff
@@ -230,10 +230,10 @@ String GDMonoMethod::get_signature_desc(bool p_namespaces) const {
 }
 
 void GDMonoMethod::get_parameter_names(Vector<StringName> &names) const {
-	const char *_names[params_count];
-	mono_method_get_param_names(mono_method, _names);
+	const char* _names = new char[params_count];
```

Comment:
Shouldn't this use the Godot's memory allocation functions? It also needs to be freed at the end of the function.

Reply:
@vnen Fixed, thanks for pointing that out :)